packages/fast-usdc/src/cli/lp-commands.js

```diff
@@ -50,7 +55,7 @@ const parseUSDCAmount = (amountString, usdc) => {
  */
 export const addLPCommands = (
   program,
-  { fetch, vstorageKit, stderr, stdout, env, now },
+  { fetch, agoricNames, vstorageKit, stderr, stdout, env, now },
```

Comment:
It seems a little fickle to inject both `agoricNames` and `vstorageKit` because `agoricNames` should always come from the `vstorageKit`'s queries. Being able to independently craft them allows them to have conflicting results. I.e. `vstorageKit` could return something when querying `published.agoricNames.brands` that doesn't match the `agoricNames` arg. It would be helpful to at least document these options as test-only to avoid confusion. So request: please add a doc/comment saying these are for testing.

Reply:
> It seems a little fickle to inject both agoricNames and vstorageKit because agoricNames should always come from the vstorageKit's queries

Not anymore, right? The PR is about removing `agoricNames` from `VstorageKit`.

Reply:
I thought this PR is about making vstorageKit synchronous. If there's a context where looking at a provided `agoricNames` object contradicts what would result by calling the provided `vstorageKit.readPublished("agoricNames...")` then I think that's a bad consequence. Perhaps make `vstorageKit.agoricNames` a promise instead of removing it then?

Reply:
> Perhaps make vstorageKit.agoricNames a promise instead of removing it then?

I don't think it's necessary to do this. My main concern is that we have a function with two arguments that can be two different sources of truth for the same data, it would just be nice to highlight this hazard somehow with a comment.

Reply:
>My main concern is that we have a function with two arguments that can be two different sources of truth for the same data, it would just be nice to highlight this hazard somehow with a comment.

Oh, I see. Yes that is a hazard. I'll work on that.

Reply:
b7180b007deaf0be90a4a8afb5fc44ab0fc9f390